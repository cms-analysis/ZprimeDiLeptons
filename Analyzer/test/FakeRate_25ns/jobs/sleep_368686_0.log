000 (368686.000.000) 09/05 19:57:12 Job submitted from host: <90.147.169.224:9618?addrs=90.147.169.224-9618&noUDP&sock=2441165_0900_3>
...
001 (368686.000.000) 09/05 20:15:52 Job executing on host: <90.147.168.234:42263>
...
006 (368686.000.000) 09/05 20:16:01 Image size of job updated: 125808
	79  -  MemoryUsage of job (MB)
	80840  -  ResidentSetSize of job (KB)
...
005 (368686.000.000) 09/05 20:17:42 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:01:11, Sys 0 00:00:16  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:01:11, Sys 0 00:00:16  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	98278  -  Run Bytes Sent By Job
	69544  -  Run Bytes Received By Job
	98278  -  Total Bytes Sent By Job
	69544  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated
	   Cpus                 :                 1         1
	   Disk (KB)            :      429       75   2646092
	   Memory (MB)          :       79     2500      2500
...
