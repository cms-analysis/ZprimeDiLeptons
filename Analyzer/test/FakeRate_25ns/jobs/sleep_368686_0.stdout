Running ZprimeMuMu Analysis_FR with executables RunZprimeMuMuAnalysis_FR
Working dir is /home/condor/dir_7093
Saving dir is /lustre/cms/store/user/defilip/ZprimeAnalysis/80X/histos/histosZprimeMuMu_FR
Compiling the macros
